README.md

```diff
@@ -2,16 +2,16 @@
 
 [![Build Status](https://travis-ci.org/igneus/church-calendar-api.svg?branch=master)](https://travis-ci.org/igneus/church-calendar-api)
 
-Browse Roman Catholic liturgical calendar through a web interface
-or obtain it's data in JSON format through an API.
+Browse the Roman Catholic liturgical calendar through a web interface
+or obtain its data in JSON format through an API.
 
 Powered by
 [calendarium-romanum][caro] and
 grape.
 
-## Explore it on-line
+## Explore it online
 
-Church Calendar API is a more or less RESTful read-only JSON API.
+Church Calendar API is more or less a RESTful read-only JSON API.
```

Comment:
Another minor point. Colloquially, the placement of 'a' could come before or after the expression more or less. Rereading it a second time, the only thing I would modify grammar-wise is adding a comma, but I don't think that would be necessary for a README.

Reply:
I must admit it's been 10+ years since I was subject to thorough English language drills for the last time, so I may be wrong here, but I don't think the meaning of the two text versions is identical:

> Church Calendar API is **a** *more or less* RESTful read-only JSON API.

Here the adverbs "more or less" modify just the adjective "RESTful", which is exactly what I wanted to say: the RESTful nature of the API could be disputed and I'm aware of that.

> Church Calendar API is *more or less* **a** RESTful read-only JSON API.

Here the adverbs "more or less" modify the whole expression "RESTful read-only JSON API", which is not what I wanted to say. Church Calendar API is a "read-only JSON API" - not just "more or less", but 100%. It's just "more or less" RESTful.

Reply:
I see, then with that context, I withdraw my proposed change for this line.